MN/Algoritmi/Metode-Iterative/SOR/SOR.m

```diff
@@ -0,0 +1,31 @@
+%  Solves linear systems A*x = B using super-over-relaxation method
+%  success - 0 = solution found / 1 = maxiter exceeded
```

Comment:
vreau sa eliminam succes, dar e contraintuitiv sa pui succes = 0 la found si 1 la not found